chronos/asyncloop.nim

```diff
@@ -729,6 +747,28 @@ elif unixPlatform:
     # poll() call.
     loop.processCallbacks()
 
+    # Wait until we have a decent amount of data and pick the most frequently
+    # seen futures.
+    const
```

Comment:
I must agree this feature needs more compile-time configuration and perhaps it should be disabled by default for a library such as Chronos.